src/Frontend/Core/Header/Header.php

```diff
@@ -351,14 +352,36 @@ public function parse(): void
         $this->cssFiles->parse($this->template, 'cssFiles');
         $this->jsFiles->parse($this->template, 'jsFiles');
 
-        $siteHTMLHeader = (string) $this->get('fork.settings')->get('Core', 'site_html_header', '') . "\n";
-        $siteHTMLHeader .= new GoogleAnalytics(
-            $this->get('fork.settings'),
-            Model::getRequest()->getHttpHost(),
-            $this->get('fork.cookie')
-        );
-        $siteHTMLHeader .= "\n" . $this->jsData;
-        $this->template->assignGlobal('siteHTMLHeader', trim($siteHTMLHeader));
```

Comment:
for now I would assign it to the both of them for BC reasons. There are people that upgrade to the latest version of fork and this will break the theme that they use

Reply:
Fixed in dabf936ae918db1eac41e60b3cfe98587fc8d0aa